packages/asc-ui/src/utils/deprecatedWarning.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * A general deprecated console warning avoided by lint
+ */
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
We can use a default export here since it's a single function file.

Reply:
👍